grunt/tasks/watch.js

```diff
@@ -3,8 +3,20 @@
  *
  */
 module.exports = {
-  task: function() {
+  task: function () {
     return {
+      jasmine: {
```

Comment:
This is great to get some instant feedback when writing/changing tests, but I personally find it not so useful when doing something else (eg: changing tests that are not executed by the `npm-test` task). It makes things a bit slower (even if tests are very fast). I see there's also `npm run test-watch`, so perhaps we can remove this?